charm/sisyphus/actions.yaml

```diff
@@ -0,0 +1,16 @@
+start:
+  description: Starts sisyphus
+  params:
```

Comment:
I think these could be config options. I don't see much need to change these very often, and the action would be easier to run with no params. Thoughts?